pallets/collator-assignment/src/lib.rs

```diff
@@ -236,6 +244,27 @@ pub mod pallet {
                     max_collators: collators_per_parathread,
                 });
             }
+
+            // Are there enough collators to satisfy the minimum demand?
+            let enough_collators_for_all_chain = collators.len() as u32
+                > T::HostConfiguration::min_collators_for_orchestrator(target_session_index)
```

Comment:
```suggestion
                >= T::HostConfiguration::min_collators_for_orchestrator(target_session_index)
```

If the number of collators is equal to the required we also don't want to apply the congestion fee. Not sure if this was correct before the last refactor.

Reply:
Good catch!